### PR TITLE
Added React JSX definition

### PR DESCRIPTION
**Pull request by vkbansal**
capture

Support for React JSX http://facebook.github.io/react/

### CODE REVIEW

**Comment by Golmote** [comment]
(This fixes #391)

**Comment by LeaVerou** [comment]
LGTM, merged, thanks!

**Comment by Golmote** [comment]
Please welcome our first multi-dependencies language! \o/ d380942a12e2f35b023e43a18007d47cf9a57be2

File: components/prism-jsx.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Prism.languages.insertBefore('inside', 'attr-value',{
+	'script': {
+		pattern: /=(\{[\w\W]*?\})/gi,
+		inside: {
+			'function' : Prism.languages.javascript.function,
+			'punctuation': /[={}[\];(),.:]/,
+			'keyword':  Prism.languages.javascript.keyword,
+		},
+		'alias': 'javascript'
```
Comment (vkbansal) [inline components/prism-jsx.js]: @Golmote can you change alias to 'language-javascript' ?

**Comment by LeaVerou** [comment]
> Please welcome our first multi-dependencies language! \o/

Hehe yeah, I noticed that! Woohoo! :)

File: components/prism-jsx.js (hunk 1 above)
Comment (Golmote) [inline components/prism-jsx.js]: @vkbansal Sure. e444b56ae65c3815689b167ff272e9b36e9de3de

**Pull request merged**